src/examples/shaping_tutorial/lesson_00/README.md

```diff
@@ -0,0 +1,234 @@
+
+Shaping produces "volume fraction" fields defined on mesh cells, representing the percentage of each cell occupied by different materials. In high-order finite element meshes, these volume fractions are polynomial functions describing material distribution throughout the cells. Volume fractions maintain a partition of unity property, meaning they always sum to one at any location in the mesh, ensuring complete and consistent material representation.
+
+<br />
+
+<figure style="text-align: center;">
+  <img src="shaping_workflow.png" alt="Shaping workflow in Axom" style="display: inline-block;" />
+  <figcaption>Figure: Overview of the shaping workflow in this tutorial. Geometric regions are combined with a computational mesh to produce volume fractions</figcaption>
+</figure>
+
+## Key Axom components in this tutorial
+
+- **Klee**: Plays a critical role in flexible geometric setup for multimaterial simulations. It utilizes **Inlet** (a parameterized input library) and **Sidre** (a data management interface), allowing users to define their problem settup effectively.
+
+- **Quest**: The application framework used in shaping. Quest leverages the **Primal** and **Spin** components, facilitating the actual shaping computations and volume fraction assignments.
+
+
+```mermaid
+%%{init: {
+  "theme": "base",
+  "themeVariables": {
+    "primaryColor": "#e6f0ff",
+    "primaryBorderColor": "#3366cc",
+    "primaryTextColor": "#000000",
+    "lineColor": "#3366cc",
+    "background": "#ffffff",
+    "fontFamily": "Verdana"
+  }
+}}%%
+graph LR
+  classDef highlight fill: #ffe082, stroke: #e65100, stroke-width: 3px;
+
+  bump --> spin
+  inlet:::highlight --> primal
+  inlet --> sidre
+  klee:::highlight --> inlet
+  lumberjack --> core
+  mint --> slam
+  mir --> bump
+  multimat --> slam
+  sidre:::highlight --> slic
+  primal --> slic
+  quest:::highlight --> spin
+  quest --> mint
+  sina --> core
+  slam --> core
+  spin --> slam
+  spin --> primal
+
+  %% dashed (soft deps)
+  mint -.-> sidre
+  quest -.-> klee
+  slic -.-> lumberjack
+
+  primal --> core
+  sidre --> core
+  slam --> slic
+
+  %% links can be highlighted as follows:
+  %% linkStyle 5 stroke: #d32f2f,color: #d32f2f, stroke-width:3px;
+```
+<div style="text-align:center">
+<figcaption>Figure: Axom components and their interdependencies, highlighting the components featured in this tutorial</figcaption>
+</div>
+
+## A first example
+
+Let's get our feet wet with a first example that uses an installed version of Axom. Specifically, we will print the version of Axom and some of its configuration properties.
+
+> :information_source: Our examples use the ``C++17`` standard since that's Axom's current version.
+
+
+> :memo: Note for HPCIC tutorial on <time datetime="2025-09-16">16 Sep 2025</time>: This tutorial currently lives on the `feature/shaping-tutorial` branch of Axom. To use it, check out that branch, then build and install Axom. 
```

Comment:
Are you going to leave this branch around long-term then?

Reply:
I'm planning to remove that note. I added it the day before since the image didn't have a compiled version of axom. In the future, I'd plan to have the tutorial installed so the audience won't need to worry about compiling.

Reply:
I removed this line.